packages/dds/tree/src/util/multimap.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Multimap<K, V> extends Map<K, V[]> {
+    add(key: K, value: V): void {
+        this.get(key).push(value);
```

Comment:
Don't you need to check for the case where there is no entry for the key? (and if so, populate it)

Reply:
The entry is created by `get`.

Reply:
Oh sorry, I missed the override below. -_-"